src/services.py

```diff
@@ -0,0 +1,56 @@
+from __future__ import print_function
+
+from django.conf import settings
+
+from nameko.events import EventDispatcher
+from nameko.rpc import rpc
+
+
+class ServiceHandler(object):
+
+    def __init__(self, service):
+        self.service = service
+
+    def send(self, content):
+        """Send content over RPC to a service, triggering pre and post actions.
+
+        Args:
+            content (object): Content to send over RPC to connected service.
+        """
+        prepared_content = self.service.pre_send(content)
+
+        try:
+            with settings.CLUSTER_RPC as cluster_rpc:
+                running_service = getattr(cluster_rpc, self.service.name)
+                running_service.send(prepared_content)
+        except Exception as e:
```

Comment:
Is catching the exception necessary? Unless you don't want the program to stop here?